modules/dbparser/tests/test_parsers_e2e.c

```diff
@@ -57,9 +57,9 @@ _assert_pattern(PatternDB *patterndb, const gchar *pattern, const gchar *rule, g
   gboolean result;
   LogMessage *msg = log_msg_new_empty();
 
-  log_msg_set_value(msg, LM_V_HOST, MYHOST, strlen(MYHOST));
-  log_msg_set_value(msg, LM_V_PROGRAM, "test", strlen(MYHOST));
-  log_msg_set_value(msg, LM_V_MESSAGE, pattern, strlen(pattern));
+  log_msg_set_value(msg, LM_V_HOST, MYHOST, -1);
```

Comment:
Is there an actual problem with the original version, or this patch is a refactor? log_msg_set_value seems to do the same (calculating strlen) when negative value is passed.

Reply:
Yes, the second line sets `"test"` and uses `strlen(MYHOST)`.

Reply:
hmmm ok :) I did not catch it.